README.md

```diff
@@ -72,6 +72,10 @@ updating the frontend.
 
 * Create a new [user](https://www.keycloak.org/docs/latest/getting_started/index.html#_create-new-user) in the tdr realm.
 
+* Set up another [client](https://www.keycloak.org/docs/latest/server_admin/#oidc-clients) called tdr-fe.
```

Comment:
Can you mention what type of client it is? It won't be a confidential client like the server-side one.

Reply:
Ok, I've added a line in explaining it's a public client.